pkg/session_recording/recordings.jsx

```diff
@@ -85,6 +87,32 @@
         return (ms < 0 ? '-' : '') + str;
     };
 
+    let isDateValid = function(date) {
+        let regex = new RegExp(/^\s*(\d\d\d\d-\d\d-\d\d)(\s+(\d\d:\d\d(:\d\d)?))?\s*$/,["i"]);
```

Comment:
We don't need the `i` flag here, as there are no letters in the pattern, please remove.